output.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include<iostream>
+#include<fstream>
+#include<vector>
+#include<iomanip>
+using namespace std;
+
+void outputTextFile(vector <pair<string, string>> x)
+{
+  ofstream outputFile;
+  outputFile.open("output.txt");
+  outputFile << "       Output:       " << endl;
+  outputFile << "Token          Lexeme" << endl;
+  outputFile << "---------------------" << endl;
+  for (int i = 0; i < x.size(); ++i)
+  {
+    outputFile << x[i].first << setw(12) << x[i].second << endl;
```

Comment:
I was able to fix the formatting problem you were talking about earlier. 
If you replace line 17 with this line of code it should work.
`outputFile << x[i].first << std::right << setw(20 - x[i].first.length()) << x[i].second << std::endl;`